version: "3"

services:

  web:
    image: "plexinc/pms-docker:plexpass"
    environment:
      - "HOSTNAME={{ plex_hostname }}"
      - "TZ={{ plex_tz }}"
      - "ADVERTISE_IP={{ plex_ip }}"
      - "PLEX_CLAIM={{ plex_claim }}"
    deploy:
      restart_policy:
        condition: "on-failure"
      labels:
        - "traefik.enable=true"
        - "traefik.backend={{ role_path | basename }}"
        - "traefik.port=32400"
        - "traefik.docker.network={{ docker_proxy_network }}"
        - "traefik.frontend.rule=Host:{{ plex_hostname }}"
    networks:
      - "{{ docker_proxy_network }}"
    volumes:
      - "/volume1/{{ inventory_hostname[4:7] | upper() }}Stack/{{ role_path | basename }}/config:/config"
      - "/volume1/{{ inventory_hostname[4:7] | upper() }}Data/{{ role_path | basename }}/transcode:/transcode"
      - "/volume1/{{ inventory_hostname[4:7] | upper() }}Data/btn/pub:/var/pub/television:ro"
      - "/volume1/{{ inventory_hostname[4:7] | upper() }}Data/ptp/pub:/var/pub/movie:ro"

networks:
  {{ docker_proxy_network }}:
    external: true
